netcdf/variables.go

```diff
@@ -81,7 +81,10 @@ func (v Var) LenDims() ([]uint64, error) {
 // NAttrs returns the number of attributes assigned to variable v.
 func (v Var) NAttrs() (n int, err error) {
 	var cn C.int
-	err = newError(C.nc_inq_natts(C.int(v.ds), &cn))
+	// nc_inq_natts return the number of global attributes
```

Comment:
This comment is too verbose. I'd just remove the comment altogether, since NetCDF C API is very well documented already.